src/Keyboard.js

```diff
@@ -58,3 +59,12 @@ module.exports = {
     KEY_Y: 89,
     KEY_Z: 90,
 };
+
+export function isCtrlOrCmdKeyEvent(ev) {
```

Comment:
might be nice to call this isOnlyCtrlOrCmdKeyEvent or something to spell out that it's explicitly not going to match shift-cmd or meta-cmd etc.

Reply:
Yep, agreed